ups/ts_guitool.table

```diff
@@ -0,0 +1,11 @@
+# List EUPS dependencies of this package here.
+# - Any package whose API is used directly should be listed explicitly.
+# - Common third-party packages can be assumed to be recursively included by
+#   the "sconsUtils" package.
+setupRequired(sconsUtils)
```

Comment:
This line is not required.

Reply:
Updated in the commit: [c2950b7](https://github.com/lsst-ts/ts_guitool/pull/1/commits/c2950b7e377381a298a81fd240ce595db475b8d9). Thanks!